src/components/Converter.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div id="converter">
+    <p>{{ label }}</p>
+    <input v-model.number="convert" type="number" placeholder="Enter USD" />
```

Comment:
placeholder is hardcoded, should be a prop that's different depending on the converter